src/app/tests/suites/commands/system/SystemCommands.cpp

```diff
@@ -26,49 +26,60 @@ const char * getScriptsFolder()
 }
 } // namespace
 
-constexpr size_t kCommandMaxLen = 256;
+constexpr size_t kCommandMaxLen    = 256;
+constexpr const char * kDefaultKey = "default";
 
-CHIP_ERROR SystemCommands::Start(uint16_t discriminator, uint16_t port, const char * kvs, const char * registerKey)
+CHIP_ERROR SystemCommands::Start(const char * identity, const chip::app::Clusters::SystemCommands::Commands::Start::Type & value)
 {
     const char * scriptDir            = getScriptsFolder();
     constexpr const char * scriptName = "Start.py";
 
     char command[kCommandMaxLen];
     chip::StringBuilderBase builder(command, sizeof(command));
-    ReturnErrorOnFailure(CreateCommonCommandArgs(builder, scriptDir, scriptName, registerKey, discriminator, port, kvs));
+    ReturnErrorOnFailure(CreateCommonCommandArgs(builder, scriptDir, scriptName, value));
```

Comment:
FYI, with `chip::app::Clusters::SystemCommands::Commands::Start::Type` being passed in now, `CreateCommonCommandArgs` is no longer needed. My original planned use was to create a `SystemCommands::StartOtaProvider` that looks just like `SystemCommands::Start` which will call `CreateCommonCommandArgs` and then add any additional OTA-specific params via the builder afterwards. With `chip::app::Clusters::SystemCommands::Commands::Start::Type`, which encapsulates all the types, I no longer need to add `SystemCommands::StartOtaProvider`.

Reply:
I removed the method.